src/Keboola/DbExtractor/Extractor/Common.php

```diff
@@ -32,12 +32,21 @@ public function createConnection($params)
         $pdo = new \PDO($dsn, $params['user'], $params['password'], $options);
         $pdo->setAttribute(\PDO::MYSQL_ATTR_USE_BUFFERED_QUERY, false);
         $pdo->exec("SET NAMES utf8;");
-
         return $pdo;
     }
 
     public function testConnection()
     {
         $this->db->query("SELECT 1");
     }
+
+    public function showTables()
+    {
+        $tables = [];
+        $res = $this->db->query("SHOW TABLES");
+        while ($table = $res->fetch(\PDO::FETCH_NUM)) {
```

Comment:
woudln't be more safe for feature extensions return array of objects?
something like:
```
[
 {
  "name": "escaping"
 },
 {
  "name": "another"
 }
]
```